### ISSUE TITLE
Allow user to filter the list of styles

### ISSUE THREAD

**Issue by thisisfranciswu** [3-shake/figma-style-finder#3]
I'm attempting to swap library files but unfortunately there's no easy way to resolve orphaned styles in the new library, as seen here:

![image](https://user-images.githubusercontent.com/13093/216686201-6b8f2723-11d1-41f6-ba97-1dffcd0ac7e2.png)

Your plugin has come closest to solving this problem, which I appreciate enough to [buy you a coffee](https://www.buymeacoffee.com/) :).

Aside from the request to [allow users to refresh the list of styles](https://github.com/3-shake/figma-style-finder/issues/1), it'd be great to be able to filter the list so that I can more easily resolve orphaned styles.

Thanks!

**Comment by Qs-F** [comment]
@thisisfranciswu

Thanks for your suggestion!
If you have any other recommendation for #1 (reloading function), please write on the issue!

For the filter function, sorry but I couldn't get your point.
Is that something like filtering by text? (For example, suppose you have styles "aaa", "abc" and "bbb", and if you type "a" to a textfield, only "aaa" and "abc" are shown)

Or, more like complex one?

**Comment by thisisfranciswu** [comment]
Yes, exactly that! Just filtering by style name for starters. Perhaps make it case-insensitive too.

**Comment by Qs-F** [comment]
@thisisfranciswu Got it! I'll try

**Comment by thisisfranciswu** [comment]
Thanks! Also, I'm serious about the coffee :).
